Update README.md for nubies like me (#14)
space-wizards/SS14.Changelog: README.md

  GitHubSecret: "<SECRET>"
  # The branch to look at for generating changelogs.
  ChangelogBranchName: "master"

>   # File in Resources/Changelog/ where they will write

Reviewer: ```suggestion
  # File in Resources/Changelog/ that will be updated.
```
